app/pages/api/apply/[page].js

```diff
@@ -19,9 +20,15 @@ async function handler(req, res) {
   session.formData = newFormData;
 
   const context = { req, newData: newFormData, page, js };
-
   if (page === LAST_PAGE) {
-    await submitApplication(context);
+    let success = await submitApplication(context);
+    if (success !== undefined) {
+      let [result, res, userEmail] = success;
```

Comment:
we have `req` object above `const context = { req, newData: newFormData, page, js };`
and can find `res` object within the `req` objects as well.
so I believe we might not need to change the return values in `submitApplication` function.